plugin/nvim-treesitter.vim

```diff
@@ -41,6 +41,7 @@ highlight default link TSProperty Identifier
 highlight default link TSConstructor Special
 highlight default link TSAnnotation PreProc
 highlight default link TSAttribute PreProc
+highlight default link TSNamespace Constant
```

Comment:
Why adding twice?

Reply:
maybe that was why it was still a draft :-)